packages/ui/components/topbanner/index.ts

```diff
@@ -0,0 +1 @@
+export * from "./TopBanner";
```

Comment:
I'd rather be more explicit on what we're actually importing here.
```suggestion
export { TopBanner, TopBannerProps } from "./TopBanner";
```